config/initializers/hesa_degree_types.rb

```diff
@@ -1,92 +1,86 @@
-  [67, "Bachelor of Fine Art(BFA)"],
-  [68, "Bachelor of General Studies(BGS)"],
-  [69, "Bachelor of Humanities (BH)"],
-  [70, "Bachelor of Hygiene (B HYG)"],
-  [71, "Bachelor of Law(LLB)"],
-  [72, "Bachelor of Librarianship (B LIB)"],
-  [73, "Bachelor of Librarianship and Info. Studies(BLS)"],
-  [74, "Bachelor of Literature (B LITT)"],
-  [75, "Bachelor of Land Economy(BL ECON)"],
-  [76, "Bachelor of Medical Science(B MED SC)"],
-  [77, "Bachelor of Medicine(BM)"],
-  [78, "Bachelor of Metallurgy (B MET)"],
-  [79, "Bachelor of Metallurgy and Engineering(B MET/EN)"],
-  [80, "Bachelor of Music (B MU)"],
-  [81, "Bachelor of Nursing (BN)"],
-  [82, "Bachelor of Pharmacy(B PHARM)"],
-  [83, "Bachelor of Science (B SC)"],
-  [84, "Bachelor of Science Economics (BSC/ECO)"],
-  [85, "Bachelor of Science &amp; Engineering (BSC/ENG)"],
-  [86, "Bachelor of Science &amp; Technology (BSC/TEC)"],
-  [87, "Bachelor of Science in Social Science (BSC SS)"],
-  [88, "Bachelor of Science in Speech Therapy (BSC SPT)"],
-  [89, "Bachelor of Social Science (BSS)"],
-  [90, "Bachelor of Surgery (BS)"],
-  [91, "Bachelor of Chirurgiae (B CH)"],
-  [92, "Bachelor of Technology (B TECH)"],
-  [93, "Bachelor of Theology (B THEOL)"],
-  [94, "Bachelor of Veterinary Medicine (B VMED)"],
-  [95, "Bachelor of Veterinary Medicine and Surgery (BVMS)"],
-  [96, "Bachelor of Veterinary Science (B VSC)"],
-  [97, "Bachelor of Education Scotland &amp; N Ireland (B ED)"],
-  [98, "Bachelor of Philosophy (B PHIL)"],
-  [200, "Master of Arts (MA)"],
-  [201, "Master of Librarianship (M LIB)"],
-  [202, "Master of Literature (M LITT)"],
-  [203, "Master of Music (M MUS)"],
-  [204, "Master of Philosophy (M PHIL)"],
-  [205, "Master of Science (M SC)"],
-  [206, "Master of Theology (M THEOL)"],
-  [207, "Certificate of Membership of Cranfield Institute of Technology"],
-  [208, "Master of Education (M ED)"],
-  [209, "Master of Business Studies (M BS)"],
-  [210, "Master of Social Studies (M SS)"],
-  [211, "Master of Engineering (M ENG)"],
-  [212, "Master of Law (M LAW)"],
-  [213, "Master of Business Administration (MBA)"],
-  [214, "Master of Chemistry (M CHEM)"],
-  [215, "Master of Physics (M PHYS)"],
-  [300, "Doctor of Divinity (DD)"],
-  [301, "Doctor of Civil Law (DCL)"],
-  [302, "Doctor of Medicine (MD)"],
-  [303, "Doctor of Music (D MU)"],
-  [304, "Doctor of Science (D SC)"],
-  [305, "Doctor of Philosophy (D PHIL)"],
-  [306, "PhD (PHD)"],
-  [400, "First Degree"],
-  [401, "Higher Degree"],
-  [402, "Degree equivalent (inc. foreign qualifications)"],
-  [999, "Unknown"],
+  [1, nil, 'BEd', :bachelor],
+  [3, nil, 'BSc/Education', :bachelor],
+  [5, nil, 'BTech/Education', :bachelor],
+  [7, nil, 'BA/Education', :bachelor],
+  [9, nil, 'BA Combined Studies/Education of the Deaf', :bachelor],
+  [12, nil, 'BA with intercalated PGCE', :bachelor],
+  [51, 'BA', 'Bachelor of Arts', :bachelor],
+  [52, 'BA ECON', 'Bachelor of Arts Economics', :bachelor],
```

Comment:
The structs in the prototype have the traditional casing (`BAEcon` rather than `BA ECON`). Should we use prototype-style `BAEcon`?

Reply:
Oops, I missed @paulrobertlloyd's comment when I reviewed this by moonlight 🙄